$NetBSD: patch-pager.cabal,v 1.1 2023/11/02 01:24:16 pho Exp $

Do not build or install the example executable. We don't want it in pkgsrc.

--- pager.cabal.orig	2023-11-02 01:23:05.923211243 +0000
+++ pager.cabal
@@ -56,6 +56,7 @@ library
     , transformers
   
 executable hs-pager-test-pager
+  buildable: False
   default-language:    Haskell2010
   hs-source-dirs: test
   other-modules: Paths_pager
